### PR TITLE
Separate link

### PR DESCRIPTION
**Pull request by wtachau**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -11,19 +11,24 @@ def relevant
...
 	def edit
-		@current_user = current_user
 		render partial:'form'
 	end
 
 	def show
-		@current_user = current_user
-		render partial:'full_post', locals:{ post: (Post.find params[:id]).decorate }
+		params.permit :id
+		@post = (Post.find params[:id]).decorate
```
Comment (mcafeeryan) [inline app/controllers/posts_controller.rb:19]: can condense to `@post = (Post.find params.permit(:id)).decorate`

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -11,19 +11,24 @@ def relevant
...
+		# @current_user = current_user.decorate
```
Comment (mcafeeryan) [inline app/controllers/posts_controller.rb:20]: get rid of commented out code

Comment (wtachau) [inline app/controllers/posts_controller.rb:20]: get rid of comment out code _please_

File: app/views/layouts/_header.html.slim [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	#logo_full
+		span class='logo_1' Originate
+		span class='logo_2' Spaces
+	div class='upper_right' id='add_profile'
+		img src=image_path('profile.png') data-toggle="modal" data-target="#profilePopup"
+	div class='upper_right' id='add_post'
+		img src=image_path('plus.png') data-toggle="modal" data-target="#addPostPopup"
+
+// Add Post modal
+#addPostPopup class='modal fade popup' tabindex='-1' role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"
```
Comment (mcafeeryan) [inline app/views/layouts/_header.html.slim]: generally front end best practices say to avoid ids and use classes instead.

**Comment by mcafeeryan** [comment]
mostly lgtm

File: app/views/layouts/_header.html.slim (hunk 3 above)
Comment (wtachau) [inline app/views/layouts/_header.html.slim]: I agree -- in this case it's useful, since it's a bootstrap modal http://getbootstrap.com/javascript/#modals

**Pull request merged**